src/test/java/org/dinosaur/foodbowl/domain/follow/presentation/FollowControllerTest.java

```diff
@@ -241,6 +242,131 @@ class 팔로워_목록_조회 {
         }
     }
 
+    @Nested
+    class 다른_회원_팔로워_목록_조회 {
+
+        @Test
+        void 팔로워_목록을_조회하면_200_응답을_반환한다() throws Exception {
+            mockingAuthMemberInResolver();
+
+            PageResponse<OtherUserFollowerResponse> response = new PageResponse<>(
+                    List.of(new OtherUserFollowerResponse(
+                            1L,
+                            "http://justdoeat.shop/static/images/profile.png",
+                            "coby5502",
+                            20,
+                            true
+                    )),
+                    true,
+                    true,
+                    false,
+                    0,
+                    1
+            );
+            given(followService.getOtherUserFollowers(anyLong(), anyInt(), anyInt(), any(Member.class)))
+                    .willReturn(response);
+
+            MvcResult mvcResult = mockMvc.perform(get("/v1/follows/{memberId}/followers", 1L)
+                            .header(AUTHORIZATION, BEARER + jwtTokenProvider.createAccessToken(1L, ROLE_회원))
+                            .param("page", "1")
+                            .param("size", "1"))
+                    .andDo(print())
+                    .andExpect(status().isOk())
+                    .andReturn();
+
+            String jsonResponse = mvcResult.getResponse().getContentAsString();
+            PageResponse<OtherUserFollowerResponse> result = objectMapper.readValue(
+                    jsonResponse,
+                    new TypeReference<PageResponse<OtherUserFollowerResponse>>() {
+                    }
```

Comment:
이 부분 인텔리제이에서 아래의 코드로 제시하던데, 명시해주신 이유가 궁금해요 ~
```suggestion
                    new TypeReference<>() {
                    }
```

Reply:
앞의 `PageResponse<FollowingResponse>`로 인해 타입 추론이 가능해서 명시않아도 된다고 인텔리제이에서 설명해주고 있는 것 같아요! 약간 `List<String> strings = new ArrayList<>();` 요런 것처럼요 ㅎㅎ

명시하면 좋지만, 위의 상황에서도 대부분 뒤의 괄호에는 명시하지 않으므로 제거하도록 하였습니다 :)

Reply:
**1번** 
```java
PageResponse<FollowingResponse> result = objectMapper.readValue(jsonResponse, new TypeReference<>() {
});
```

**2번**
```java
PageResponse<FollowingResponse> result = objectMapper.readValue(
        jsonResponse,
        new TypeReference<>() {
        }
);
```

1번과 2번 중 컨벤션 추천 부탁드립니다! 설정해둔 컨벤션 때문에 `{}` 괄호 1개가 자꾸 아래로 내려가요 🥲

Reply:
2번으로 가도 크게 문제 없을 것 같아요 ㅎㅎ